Makefile

```diff
@@ -6,7 +6,7 @@ TARGET ?= Kickstarter-Framework
 PLATFORM ?= iOS
 RELEASE ?= itunes
 # Keep simulator in sync with `Library/TestHelpers/TestCase.swift` and `.circleci/config.yml`
-IOS_VERSION ?= 17.2
+IOS_VERSION ?= 17.5
```

Comment:
Is there any decision made by the core team to drop support for iOS 17.2 ?

Reply:
What do you mean by drop support? We should definitely have the app still work for users on iOS 17.2. However, if we're updating our circleCI config, my recommendation is to have developers match that configuration locally.

Reply:
plus the @ifosli replied, the iOS 17.5 is the minimum runtime version available in CircleCI->Xcode 16 

more info https://circle-macos-docs.s3.amazonaws.com/image-manifest/v15048/manifest.txt

Reply:
Thanks for clarification.